kncaputo/underrated
# Testing

## What's this PR do?
- Creates tests for all apiCalls
- Adds render test for `StarRating`, `MovieDetails`, `MovieGrid` and `Poster` components
- Adds `mockMovies` sample data

## Where should the reviewer start?
- All listed files above

## How should this be manually tested?
- Run `npm test` in the terminal to see all passing tests
- Run the app and ensure all functionality is there
- Closes #3 
- Closes #6 
- Closes #35

## Screenshots/GIFs
<img width="263" alt="Screen Shot 2021-03-23 at 11 06 59 AM" src="https://user-images.githubusercontent.com/58453566/112187660-e87a1600-8bc7-11eb-96d6-67f2f9926207.png">
